### ISSUE TITLE
UK 	United Kingdom 

### ISSUE THREAD

**Issue by DavidViral** [CSSEGISandData/COVID-19#692]
UK is not a province of the country United Kingdom

**Comment by yetzt** [comment]
No, but the United Kingdom is split in different areas, such as Gibraltar, some channel islands and – most important – the whole rest that is for simplicitys sake called United Kingdom.

**Comment by DavidViral** [comment]
So duplicating the country name in the province field is to indicate there will be only one entry for the country - no subdivisions?

**Comment by JiPiBi** [comment]
If you look at France's  lines it's the same : France as Province for the mainland and when the data will be correctly labelled ;-(    one line by territory (like Martinique, Guadeloupe, French Guyana ...) 
I had suggested to use Mainland France for the Province column but not chosen (for the moment ?) 
Perhaps in the future , it will be interesting for you to have England, Wales, Scotland.... as provinces

**Comment by neuroactive** [comment]
Duplicating the country name in the region field to represent the "mainland" is just an ugly hack. Denmark has a similar issue. If you are going to include a region field, it should contain accurate data.

**Comment by JiPiBi** [comment]
@neuroactive 
You can try to make them change 
Good luck :-)

**Comment by yetzt** [comment]
@neuroactive i agree. But worse than an an ugly hack is changing the convention and breaking stuff downstream.
